### PR TITLE
Update 092022

### PR DESCRIPTION
**Pull request by charneff**
Added clarification around related JSON data being passed to front end rather than relying on front end state filtering.
Added clarification about state update after POST, PATCH, or DELETE.
Added RESTful routing language.

Please feel free to edit as always.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,8 +26,17 @@ For this project, you must:
...
   perform CRUD actions.
+- Implement proper front end state management. You should be updating state using a
+  setState function after receiving your response from a POST, PATCH, or DELETE 
+  request. You should NOT be relying on a GET request to update state. 
 - Use good OO design patterns. You should have separate classes for each of your
-  models, and create instance and class methods as necessary.
+  models, and create instance and class methods as necessary. 
+- Routes in your application (both client side and back end) should follow RESTful
+  convention.
+- Use your backend optimally. Pass JSON for related associations to the front
```
Comment (lizbur10) [inline README.md:36]: Change to "back end" for consistency. (We're far from consistent on this throughout the curriculum, but I recently saw in the style guide that it should be two words.)

File: README.md [hunk 2]
```diff
@@ -26,8 +26,17 @@ For this project, you must:
...
+  request. You should NOT be relying on a GET request to update state. 
 - Use good OO design patterns. You should have separate classes for each of your
-  models, and create instance and class methods as necessary.
+  models, and create instance and class methods as necessary. 
+- Routes in your application (both client side and back end) should follow RESTful
+  convention.
+- Use your backend optimally. Pass JSON for related associations to the front 
+  end from the back end. You should use active record methods in your controller to grab the needed data from your database and provide as JSON to the front end. You
+  should NOT be relying on filtering front end state or a separate fetch request to
+  retrieve related data.
```
Comment (lizbur10) [inline README.md:39]: Can you fix the line lengths here? It should be 80 characters max

**Review by lizbur10 — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -26,8 +26,17 @@ For this project, you must:
...
 - Build a separate React frontend application that interacts with the API to
   perform CRUD actions.
+- Implement proper front end state management. You should be updating state using a
+  setState function after receiving your response from a POST, PATCH, or DELETE 
+  request. You should NOT be relying on a GET request to update state. 
 - Use good OO design patterns. You should have separate classes for each of your
-  models, and create instance and class methods as necessary.
+  models, and create instance and class methods as necessary. 
+- Routes in your application (both client side and back end) should follow RESTful
+  convention.
```
[the remark this answers is not in the document]
Comment (charneff) [reply README.md:35]: Excellent!

File: README.md (hunk 1 above)
Comment (charneff) [reply README.md:36]: Makes sense! Totally agreed

**Comment by charneff** [comment]
All proposed changes looks great!!!

**Pull request merged**